<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <title>简历 - 马春宇</title>
  <style>
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
  </style>
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
  <link rel='stylesheet' href='https://fonts.googleapis.com/css?family=Amethysta'>
  <style type="text/css">

  body {
      font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
      width: 1100px;
      margin: auto;
      background: #FFFFFF;
      padding: 10px 10px 10px 10px;
  }

  /* Title of the resume */
  h1 {
      font-size: 55px;
      color: #757575;
      text-align:center;
      margin-bottom:15px;
  }

  /* Titles of categories */
  h2 {
      color: #2f8dbc;
  }
  /* There is a bar just before each category */
  h2:before {
      content: "";
      display: inline-block;
      margin-right:1%;
      width: 16%;
      height: 10px;
      /* This is called "rulecolor" in the ConTeXt stylesheet. */
      background-color: #2f8dbc;
  }

  div h3 {
      margin-top: 0;
      margin-bottom: 2px;
  }


  .left {
      float: left;
  }

  .right{
      float: right;

  }

  .block1{
    margin-bottom: 10px;
  }

  .block2{
    margin-bottom: 8px;
  }

  li {
      margin-bottom: 3px;
      text-align: justify;

  }

  dl {
      width: 100%;
      overflow: hidden;hidden;
      padding: 0;
      margin: 0;
  }

  dt {
      float: left;
      clear: left;
      width: 17%;
      /* adjust the width; make sure the total of both is 100% */
      padding: 0;
      margin:0;
      margin-top: 3px;
  }
  dd {
      float: left;
      width: 83%;
      /* adjust the width; make sure the total of both is 100% */
      padding: 0;
      margin:0;
      text-align: justify;
  }
  p {
      margin-top:0;
      margin-bottom:0;
  }

  ul {
      margin-top:0;
      margin-bottom:0;
  }

  blockquote {
      text-align: center
  }

  a {
      text-decoration: none;
      color: #2f8dbc;
  }

  hr {
      color: #A6A6A6;
  }

  table {
      width: 100%;
  }
  </style>
</head>
<body>
    <p style='text-align: right;'><a href="https://chunyuma.github.io/resume/cv_english.html">Translate to English</a></p>
    <h1 id="name">马春宇</h1>
    <hr />
    <table>
        <tbody>
            <tr>
                <td style="text-align: left;"><span style="font-weight: bold;">地址: </span>暂居于美国宾州</td>
                <td style="text-align: right;"><span style="font-weight: bold;">电子邮件: </span><a href="cqm5886@psu.edu" class="email">cqm5886@psu.edu</a></td>
            </tr>
            <tr>
                <td style="text-align: left;"><span style="font-weight: bold;">领英: </span><a target='_blank' href="https://www.linkedin.com/in/chunyu-ma-419257ba"> www.linkedin.com/in/chunyu-ma-419257ba</a></td>
                <td style="text-align: right;"><span style="font-weight: bold;">个人网页: </span><a target='_blank' href="https://chunyuma.github.io/">chunyuma.github.io</a></td>
            </tr>
        </tbody>
    </table>
    <hr />
    <div>
        <h2 id="education"><i class="fa fa-graduation-cap"></i> 教育经历</h2>
        <dl class='block1'>
            <dt>08/2019-至今</dt>
            <dd>
                <p style='font-size: 110%;'><strong>宾夕法尼亚州立大学 (美国)</strong></p>
                <ul>
                    <li><p>博士: <strong>生物信息学</strong></p></li>
                </ul>
            </dd>
        </dl>        
        <dl class='block1'>
            <dt>06/2015-12/2016</dt>
            <dd>
                <p style='font-size: 110%;'><strong>圣路易斯华盛顿大学 (美国)</strong></p>
                <ul>
                    <li><p>研究生: <strong>生物统计学</strong></p></li>
                </ul>
            </dd>
        </dl>
        <dl class='block1'>
            <dt>09/2010-06/2014</dt>
            <dd>
                <p style='font-size: 110%;'><strong>南方医科大学 (中国)</strong></p>
                <ul>
                    <li><p>本科: <strong>生物信息学</strong></p></li>
                </ul>
            </dd>
        </dl>
    </div>
    <div>
        <h2 id="certificate"><i class="fa fa-flask"></i> 研究经历</h2>
        <dl class='block1'>
            <dt>05/2020-至今</dt>
            <dd>
                <p style='font-size: 110%;'><strong>David Koslicki's Laboratory, Penn State University, USA</strong></p>
                <div class='block2'>
                  <p style='margin-left:20px;font-size: 100%;'><em>机器学习相关研究</em></p>
                  <ul>
                    <li><p><strong>药物重定向</strong>: 基于包含数百万个节点和数千万条边的巨大生物医学知识图谱，利用GraphSage图神经模型结合随机森林预测药物重定向以及利用基于Actor-critic对抗强化学习模型对药物重定进行解释</p></li>
                    <li><p><strong>图嵌入模型研究</strong>: 研发一个基于encoder-decoder的模型整合更多图信息(包含图的拓扑结构，点类型，带有层级关系的边信息)来提高图链接预测 </p></li>
                  </ul>
                </div>
                <div class='block2'>
                  <p style='margin-left:20px;font-size: 100%;'><em>工具和应用程序开发</em></p>
                  <ul>
                    <li><p><strong>基于生物信息知识图谱的推理工具</strong>: 一个用于查询和探索不同的生物医学知识和数据从而回答生物问题的计算推理工具，这个项目是作为美國國立轉化科學促進中心(NCATS)的生物医学翻译项目的一部分</p></li>
                  </ul>
                </div>
            </dd>
        </dl>
        <dl class='block1'>
            <dt>02/2017-04/2019</dt>
            <dd>
                <p style='font-size: 110%;'><strong>临床生物标志物实验室, 埃默里大学, 美国</strong></p>
                <div class='block2'>
                  <p style='margin-left:20px;font-size: 100%;'><em>生物信息工具和应用程序开发</em></p>
                  <ul>
                    <li><p><strong>xmsPANDA</strong>: 开发一个基于R和R shiny的用于进行“一站式”代谢组学分析 (包含数据预处理和清理、特征选择、模型拟合、验证/评估和网络分析)的生物信息软件</p></li>
                    <li><p><strong>xMWAS</strong>: 开发一个基于R shiny的应用，用于对不同多组学平台的数据进行数据集成、可视化、聚类和差异网络分析</p></li>
                    <li><p><strong>PyCDB</strong>: 对高质量的营养和环境金属的植物螯合素复合物建立数据库</p></li>
                  </ul>
                </div>
                <div class='block2'>
                  <p style='margin-left:20px;font-size: 100%;'><em>临床代谢组学分析</em></p>
                  <ul>
                    <li><p><strong>AMD study</strong>: 使用机器学习模型去识别老年黄斑病变中改变的代谢物和代谢途径</p></li>
                  </ul>
                </div>
            </dd>
        </dl>
        <dl>
            <dt>06/2016-08/2016</dt>
            <dd>
                <p style='font-size: 110%;'><strong>奈特阿尔茨海默病研究中心, 圣路易斯华盛顿大学, 美国</strong></p>
                <div class='block2'>
                  <p style='margin-left:20px;font-size: 100%;'><em>临床统计分析</em></p>
                  <ul>
                    <li><p><strong>Driving cessation study</strong>: 使用生存分析模型对于阿尔茨海默病严重程度和美国老年人驾驶影响的关系研究 </p></li>
                  </ul>
                </div>
            </dd>
        </dl>
        <dl>
            <dt>10/2015-12/2016</dt>
            <dd>
                <p style='font-size: 110%;'><strong>再生医学中心, 圣路易斯华盛顿大学, 美国</strong></p>
                <div class='block2'>
                  <p style='margin-left:20px;font-size: 100%;'><em>生物信息工具和应用程序开发</em></p>
                  <ul>
                    <li><p><strong>FeatSNP</strong>: 人类单核苷酸多态性(SNP)的大脑特异性表观遗传注释交互式数据库</p></li>
                  </ul>
                </div>
                <div class='block2'>
                  <p style='margin-left:20px;font-size: 100%;'><em>WGS, RNA-seq and ChIP-seq Analysis</em></p>
                  <ul>
                    <li><p><strong>WGS analysis</strong>: 对不同生物研究项目开发对于的单核苷酸多态性分析流程</p></li>
                    <li><p><strong>RNA-seq analysis</strong>: 对不同生物研究项目开发进行RNA-seq分析</p></li>
                    <li><p><strong>ChIP-seq analysis</strong>: 通过ChIP-seq对胚胎细胞发育过程中表观遗传修饰的差异的分析</p></li>
                  </ul>
                </div>
            </dd>
        </dl>
    </div>

    <div>
        <!-- <div>
        <h2 id="conference-proceedings"><i class="fa fa-group"></i> Conference Proceedings</h2>
        <div>
            <h3 id="presentations"><strong>Presentations</strong></h3>
            <ol type="1">

            </ol>
        </div>
        <div>
            <h3 id="posters"><strong>Posters</strong></h3>
            <ol type="1">

            </ol>
        </div>

    </div> -->
        <h2 id="publications"><i class="fa fa-file-text"></i> Publications</h2>
        <div>
            <h3 id="journal-articles"><strong>Journal Articles</strong></h3>
            <ol type="1">
                <li>Sabrina L Mitchell, <strong>Chunyu Ma</strong>, William K Scott, Anita Agarwal, Margaret A Pericak-Vance, Jonathan L Haines, Dean P Jones, Karan Uppal, Milam A Brantley Jr, <a target='_blank' href="https://pubmed.ncbi.nlm.nih.gov/pubmed/34831363">Plasma Metabolomics of Intermediate and Neovascular Age-Related Macular Degeneration Patients</a>. <strong><em>Cells</em></strong>. 2021 Nov 12;10(11):3141</li>
                <li>E. C. Wood, Amy K. Glen, Lindsey G. Kvarfordt, Finn Womack, Liliana Acevedo, Timothy S. Yoon, <strong>Chunyu Ma</strong>, Veronica Flores, Meghamala Sinha, Jared C. Roach, Luis Mendoza, Andrew S. Hoffman, Eric W. Deutsch, David Koslicki, Stephen A. Ramsey, <a target='_blank' href="https://www.biorxiv.org/content/10.1101/2021.10.17.464747v2">RTX-KG2: a system for building a semantically standardized knowledge graph for translational biomedicine</a>. <strong><em>bioRxiv</em></strong>. 2021 Oct 25</li>
                <li>Xin Hu, Douglas I Walker, Yongliang Liang, Matthew Ryan Smith, Michael L Orr, Brian D Juran, <strong>Chunyu Ma</strong>, Karan Uppal, Michael Koval, Greg S Martin, David C Neujahr, Carmen J Marsit, Young-Mi Go, Kurt D Pennell, Gary W Miller, Konstantinos N Lazaridis, Dean P Jones, <a target='_blank' href="https://pubmed.ncbi.nlm.nih.gov/pubmed/34552080">A scalable workflow to characterize the human exposome</a>. <strong><em>Nature Communication</em></strong>. 2021 Sep 22;12(1):5575</li>
                <li>Ken H Liu, Mary Nellis, Karan Uppal, <strong>Chunyu Ma</strong>, ViLinh Tran, Yongliang Liang, Douglas I Walker, Dean P Jones, <a target='_blank' href="https://pubmed.ncbi.nlm.nih.gov/pubmed/32490663">Reference Standardization for Quantification and Harmonization of Large-Scale Metabolomics</a>. <strong><em>Analytical Chemistry</em></strong>. 2020 Jul 7;92(13):8836-8844</li>
                <li><strong>Chunyu Ma</strong>, Pamela Madden, Paul Gontarz, Ting Wang and Bo Zhang, <a target='_blank' href="https://pubmed.ncbi.nlm.nih.gov/pubmed/31001319">FeatSNP: An Interactive Database for Brain-Specific Epigenetic Annotation of Human SNPs</a>. <strong><em>Frontiers in Genetics</em></strong>. 2019 Feb 2;10:262</li>
                <li>Xin Hu, Joshua Chandler, Soojin Park, Ken H. Liu, Jolyn Fernandes, Michael Orr, Matthew Ryan Smith, <strong>Chunyu Ma</strong>, Sang-moo Kang, Karan Uppal, Dean Jones, Young-Mi Go, <a target='_blank' href="https://www.ncbi.nlm.nih.gov/pubmed/30529385">Low-dose cadmium disrupts mitochondrial citric acid cycle and lipid metabolism in mouse lung</a>. <strong><em>Free Radical Biology and Medicine</em></strong>. 2019 Feb 1;131:209-217</li>
                <li>Sarah H. Stout, Ganesh M. Babulal, <strong>Chunyu Ma</strong>, David B. Carr, Denise M. Head, Elizabeth A. Grant, Monique M. Williams, David M. Holtzman, Anne M. Fagan, John C. Morris, Catherine M. Roe, <a target='_blank' href="https://www.ncbi.nlm.nih.gov/pubmed/29328928">Driving cessation over a 24-year period: Dementia severity and cerebrospinal fluid biomarkers</a>. <strong><em>Alzheimer's & Dementia</em></strong>. 2018 May;14(5):610-616</li>
                <li>Matheus B. Victor, Michelle Richner, Hannah E. Olsen, Seong Won Lee, Alejandro M. Monteys, <strong>Chunyu Ma</strong>, Christine J. Huh, Bo Zhang, Beverly L. Davidson, X. William Yang & Andrew S. Yoo, <a href="https://www.ncbi.nlm.nih.gov/pubmed/29403030">Striatal neurons directly converted from Huntington's disease patient fibroblasts recapitulate age-associated disease phenotypes</a>. <strong><em>Nature Neuroscience</em></strong>. 2018 Mar;21(3):341-352</li>
                <li>Karan Uppal, <strong>Chunyu Ma</strong>, Young-Mi Go, Dean P Jones, <a  target='_blank' href="https://www.ncbi.nlm.nih.gov/pubmed/29069296">xMWAS: a data-driven integration and differential network analysis tool</a>. <strong><em>Bioinformatics</em></strong>. 2018 Feb 15;34(4):701-702</li>
                <li>Sanne Nygaard, Haofu Hu, Cai Li, Morten Schiøtt, Zhensheng Chen, Zhikai Yang, Qiaolin Xie, <strong>Chunyu Ma</strong>, Yuan Deng, Rebecca B. Dikow, Christian Rabeling, David R. Nash, William T. Wcislo, Seán G. Brady, Ted R. Schultz, Guojie Zhang & Jacobus J. Boomsma, <a target='_blank' href="https://www.ncbi.nlm.nih.gov/pubmed/27436133">Reciprocal genomic evolution in the ant-fungus agricultural symbiosis</a>. <strong><em>Nature Communication</em></strong>. 2016 Jul 20;7:12233</li>
            </ol>
        </div>
    </div>
    <div>
        <h2 id="employment"><i class="fa fa-university"></i> Internship & Employment</h2>
        <dl>
            <dt>05/2020-至今</dt><dd><strong>研究助理</strong>, David Koslicki实验室, 宾夕法尼亚州立大学, 斯泰特科利奇, 美国</dd>
            <dt>02/2017-04/2019</dt><dd><strong>生物信息学家</strong>, 临床生物标志物实验室, 埃默里大学, 亚特兰大, 美国</dd>
            <dt>06/2016-08/2016</dt><dd><strong>暑假实习生</strong>, 奈特阿尔茨海默病研究中心, 圣路易斯, 美国</dd>
            <dt>10/2015-12/2016</dt><dd><strong>研究助理</strong>, 再生医学中心, 圣路易斯华盛顿大学, 圣路易斯, 美国</dd>
            <dt>11/2013-06/2014</dt><dd><strong>生物信息学分析员</strong>, 中国国家基因库, 华大基因, 深圳, 中国</dd>
        </dl>
    </div>

    <div>
        <h2 id="proskills"><i class="fa fa-book"></i> 专业技能</h2>
        <div class='block1'>
            <h3>计算机技能</h3>
            <ul>
                <p><strong>编程语言</strong>: Python (including Cpython & Flask), C, Shell Script, R (including R shiny), SQL, Javascript, HTML, CSS</p>
                <p><strong>数据库</strong>: MySQL, MongoDB, Neo4j</p>
                <p><strong>框架</strong>: sklearn, pytorch, tensorflow, Flask</p>
                <p><strong>机器学习模型</strong>: Basic ML models (e.g. SMV, RF), Graph embedding models (e.g. TransR, RotatE, DistMult), GNN models (e.g. GCN, GAT, RGCN), RL models (e.g. Q-learning, A2C)</p>
                <p><strong>其他技能</strong>: Linux, Data Structures and Algorithms, Git, AWS, Google Could, REST API</p>
            </ul>
        </div>
        <div class='block1'>
            <h3>项目分析技能</h3>
            <ul>
                <p><strong>生物信息</strong>: RNA-seq分析, ChIP-seq分析, 代谢组学分析 (LC-MS)</p>
                <p><strong>生物统计</strong>: 生存分析</p>
            </ul>
        </div>
    </div>

    <div>
        <h2 id="software-development"><i class="fa fa-gears"></i> 应用开发产品</h2>
        <div class='block1'>
            <h3 id="contributions-to-open-source-libraries"><strong>生物信息数据库</strong></h3>
            <dl>
                <dt><a target='_blank' href="http://featsnp.org">FeatSNP</a></dt><dd>人类单核苷酸多态性(SNP)的大脑特异性表观遗传注释交互式数据库</dd>
                <dt style='margin-top: 5px;'><a target='_blank' href="https://kuppal.shinyapps.io/pycdb">PyCDB</a></dt><dd style='margin-top: 5px;'>高质量植物螯合素数据库</dd>
            </dl>
        </div>
        <div class='block1'>
            <h3 id="contributions-to-open-source-libraries"><strong>其他开源数据应用</strong></h3>
            
            <dl>
                <dt style='margin-top: 5px;'><a target='_blank' href="https://arax.ncats.io">ARAX</a></dt><dd style='margin-top: 5px;'>一个基于知识图谱，用于回答生物医学转化问题的计算推理工具</dd>
                <dt style='margin-top: 5px;'><a target='_blank' href="https://kuppal.shinyapps.io/xmwas">xMWAS</a></dt><dd style='margin-top: 5px;'>一个基于R shiny开发的用于对不同多组学平台的数据进行数据集成、可视化、聚类和差异网络分析的应用</dd>
            </dl>
        </div>
    </div>

</body>
</html>
