<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <title>Resume - Chunyu Ma</title>
  <style>
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
  </style>
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
  <link rel='stylesheet' href='https://fonts.googleapis.com/css?family=Amethysta'>
  <style type="text/css">

  body {
      font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
      width: 1100px;
      margin: auto;
      background: #FFFFFF;
      padding: 10px 10px 10px 10px;
  }

  /* Title of the resume */
  h1 {
      font-size: 55px;
      color: #757575;
      text-align:center;
      margin-bottom:15px;
  }

  /* Titles of categories */
  h2 {
      color: #2f8dbc;
  }
  /* There is a bar just before each category */
  h2:before {
      content: "";
      display: inline-block;
      margin-right:1%;
      width: 16%;
      height: 10px;
      /* This is called "rulecolor" in the ConTeXt stylesheet. */
      background-color: #2f8dbc;
  }

  div h3 {
      margin-top: 0;
      margin-bottom: 2px;
  }


  .left {
      float: left;
  }

  .right{
      float: right;

  }

  .block1{
    margin-bottom: 10px;
  }

  .block2{
    margin-bottom: 8px;
  }

  li {
      margin-bottom: 3px;
      text-align: justify;

  }

  dl {
      width: 100%;
      overflow: hidden;hidden;
      padding: 0;
      margin: 0;
  }

  dt {
      float: left;
      clear: left;
      width: 17%;
      /* adjust the width; make sure the total of both is 100% */
      padding: 0;
      margin:0;
      margin-top: 3px;
  }
  dd {
      float: left;
      width: 83%;
      /* adjust the width; make sure the total of both is 100% */
      padding: 0;
      margin:0;
      text-align: justify;
  }
  p {
      margin-top:0;
      margin-bottom:0;
  }

  ul {
      margin-top:0;
      margin-bottom:0;
  }

  blockquote {
      text-align: center
  }

  a {
      text-decoration: none;
      color: #2f8dbc;
  }

  hr {
      color: #A6A6A6;
  }

  table {
      width: 100%;
  }
  </style>
</head>
<body>
    <p style='text-align: right;'><a href="https://chunyuma.github.io/resume/cv_chinese.html">Translate to Chinese</a></p>
    <h1 id="name">Chunyu Ma</h1>
    <hr />
    <table>
        <tbody>
            <tr>
                <td style="text-align: left;"><span style="font-weight: bold;">Address: </span>Westgate Building, University Park, PA 16802</td>
                <td style="text-align: right;"><span style="font-weight: bold;">Email: </span><a href="cqm5886@psu.edu" class="email">cqm5886@psu.edu</a></td>
            </tr>
            <tr>
                <td style="text-align: left;"><span style="font-weight: bold;">Linkdin: </span><a target='_blank' href="https://www.linkedin.com/in/chunyu-ma-419257ba"> www.linkedin.com/in/chunyu-ma-419257ba</a></td>
                <td style="text-align: right;"><span style="font-weight: bold;">Personal Web: </span><a target='_blank' href="https://chunyuma.github.io/">chunyuma.github.io</a></td>
            </tr>
        </tbody>
    </table>
    <hr />
    <div>
        <h2 id="education"><i class="fa fa-graduation-cap"></i> Education</h2>
        <dl class='block1'>
            <dt>08/2019-Present</dt>
            <dd>
                <p style='font-size: 110%;'><strong>Penn State University (USA)</strong></p>
                <ul>
                    <li><p>Doctor of Philosophy in <strong>Bioinformatics</strong></p></li>
                </ul>
            </dd>
        </dl>        
        <dl class='block1'>
            <dt>06/2015-12/2016</dt>
            <dd>
                <p style='font-size: 110%;'><strong>Washington University in St. Louis (USA)</strong></p>
                <ul>
                    <li><p>Master of Science in <strong>Biostatistics</strong></p></li>
                </ul>
            </dd>
        </dl>
        <dl class='block1'>
            <dt>09/2010-06/2014</dt>
            <dd>
                <p style='font-size: 110%;'><strong>Southern Medical University (China)</strong></p>
                <ul>
                    <li><p>Bachelor of Science in <strong>Bioinformatics</strong></p></li>
                </ul>
            </dd>
        </dl>
    </div>
    <div>
        <h2 id="certificate"><i class="fa fa-flask"></i> Research Experience</h2>
        <dl class='block1'>
            <dt>05/2020-Present</dt>
            <dd>
                <p style='font-size: 110%;'><strong>David Koslicki's Laboratory, Penn State University, USA</strong></p>
                <div class='block2'>
                  <p style='margin-left:20px;font-size: 100%;'><em>Machine Learning Research</em></p>
                  <ul>
                    <li><p><strong>Drug Repurposing</strong>: Utilize GraphSage-RF model and Actor-critic adversarial reinforcement learning techniques to predict and explain the new use of drug to treat a given disease based on a huge biomedical knowledge graph which contains millions of nodes and tens of millions of edges</p></li>
                    <li><p><strong>Graph Embedding Research</strong>: Develop an encoder-decoder model to embed more graph information (e.g. graph toplogy, node types and hierarchical edge relations) into a low-dimensional continuous vector for improving the graph link prediction </p></li>
                  </ul>
                </div>
                <div class='block2'>
                  <p style='margin-left:20px;font-size: 100%;'><em>Tools and Applications Development</em></p>
                  <ul>
                    <li><p><strong>KG-based Biomedical Reasoning Tool</strong>: A computational reasoning tool (as a part of NCATS Translator Program) built for querying and exploring different biomedical knowledge and data to answer translational questions</p></li>
                  </ul>
                </div>
            </dd>
        </dl>
        <dl class='block1'>
            <dt>02/2017-04/2019</dt>
            <dd>
                <p style='font-size: 110%;'><strong>Clinical Biomarkers Laboratory, Emory University, USA</strong></p>
                <div class='block2'>
                  <p style='margin-left:20px;font-size: 100%;'><em>Bioinformatics Tools and Applications Development</em></p>
                  <ul>
                    <li><p><strong>xmsPANDA</strong>: An R package and its shiny app developed for performing “one-stop” metabolomics analysis workflow with data preprocessing and cleaning, feature selection, model fitting, validation/evaluation and network analysis</p></li>
                    <li><p><strong>xMWAS</strong>: An R package and its shiny app developed for data integration and network visualization of multiple types of omics data with statistical and graph theory techniques</p></li>
                    <li><p><strong>PyCDB</strong>: A database resource built for phytochelatin complexes of nutritional and environmental metals</p></li>
                  </ul>
                </div>
                <div class='block2'>
                  <p style='margin-left:20px;font-size: 100%;'><em>Clinical Metabolomics Analysis</em></p>
                  <ul>
                    <li><p><strong>AMD study</strong>: Utilize machine learning model to identify metabolites and metabolic pathways altered in age-related macular degeneration (AMD)</p></li>
                  </ul>
                </div>
            </dd>
        </dl>
        <dl>
            <dt>06/2016-08/2016</dt>
            <dd>
                <p style='font-size: 110%;'><strong>Knight Alzheimer’s Disease Research Center, Washington University in St. Louis, USA</strong></p>
                <div class='block2'>
                  <p style='margin-left:20px;font-size: 100%;'><em>Clinical Statistical Analysis</em></p>
                  <ul>
                    <li><p><strong>Driving cessation study</strong>: Survival analysis for exploring the relationship between driving cessation and the severity of Alzheimer’s disease based on Clinical Dementia Rating and AD's biomarkers</p></li>
                  </ul>
                </div>
            </dd>
        </dl>
        <dl>
            <dt>10/2015-12/2016</dt>
            <dd>
                <p style='font-size: 110%;'><strong>Center of Regenerative Medicine, Washington University in St. Louis, USA</strong></p>
                <div class='block2'>
                  <p style='margin-left:20px;font-size: 100%;'><em>Bioinformatics Tools and Applications Development</em></p>
                  <ul>
                    <li><p><strong>FeatSNP</strong>: An interactive database for brain-specific epigenetic annotation of human single nucleotide polymorphisms (SNPs)</p></li>
                  </ul>
                </div>
                <div class='block2'>
                  <p style='margin-left:20px;font-size: 100%;'><em>WGS, RNA-seq and ChIP-seq Analysis</em></p>
                  <ul>
                    <li><p><strong>WGS analysis</strong>: Develop an appropriate Whole Genome Sequencing (WGS) analysis pipeline for several studies</p></li>
                    <li><p><strong>RNA-seq analysis</strong>: Perform RNA-Seq data analysis to identify differentially expression genes for several studies</p></li>
                    <li><p><strong>ChIP-seq analysis</strong>: Analyze the difference in epigenetic modifications across embryonic cell development by ChIP-seq analysis</p></li>
                  </ul>
                </div>
            </dd>
        </dl>
    </div>

    <div>
        <!-- <div>
        <h2 id="conference-proceedings"><i class="fa fa-group"></i> Conference Proceedings</h2>
        <div>
            <h3 id="presentations"><strong>Presentations</strong></h3>
            <ol type="1">

            </ol>
        </div>
        <div>
            <h3 id="posters"><strong>Posters</strong></h3>
            <ol type="1">

            </ol>
        </div>

    </div> -->
        <h2 id="publications"><i class="fa fa-file-text"></i> Publications</h2>
        <div>
            <h3 id="journal-articles"><strong>Journal Articles</strong></h3>
            <ol type="1">
                <li>Sabrina L Mitchell, <strong>Chunyu Ma</strong>, William K Scott, Anita Agarwal, Margaret A Pericak-Vance, Jonathan L Haines, Dean P Jones, Karan Uppal, Milam A Brantley Jr, <a target='_blank' href="https://pubmed.ncbi.nlm.nih.gov/pubmed/34831363">Plasma Metabolomics of Intermediate and Neovascular Age-Related Macular Degeneration Patients</a>. <strong><em>Cells</em></strong>. 2021 Nov 12;10(11):3141</li>
                <li>E. C. Wood, Amy K. Glen, Lindsey G. Kvarfordt, Finn Womack, Liliana Acevedo, Timothy S. Yoon, <strong>Chunyu Ma</strong>, Veronica Flores, Meghamala Sinha, Jared C. Roach, Luis Mendoza, Andrew S. Hoffman, Eric W. Deutsch, David Koslicki, Stephen A. Ramsey, <a target='_blank' href="https://www.biorxiv.org/content/10.1101/2021.10.17.464747v2">RTX-KG2: a system for building a semantically standardized knowledge graph for translational biomedicine</a>. <strong><em>bioRxiv</em></strong>. 2021 Oct 25</li>
                <li>Xin Hu, Douglas I Walker, Yongliang Liang, Matthew Ryan Smith, Michael L Orr, Brian D Juran, <strong>Chunyu Ma</strong>, Karan Uppal, Michael Koval, Greg S Martin, David C Neujahr, Carmen J Marsit, Young-Mi Go, Kurt D Pennell, Gary W Miller, Konstantinos N Lazaridis, Dean P Jones, <a target='_blank' href="https://pubmed.ncbi.nlm.nih.gov/pubmed/34552080">A scalable workflow to characterize the human exposome</a>. <strong><em>Nature Communication</em></strong>. 2021 Sep 22;12(1):5575</li>
                <li>Ken H Liu, Mary Nellis, Karan Uppal, <strong>Chunyu Ma</strong>, ViLinh Tran, Yongliang Liang, Douglas I Walker, Dean P Jones, <a target='_blank' href="https://pubmed.ncbi.nlm.nih.gov/pubmed/32490663">Reference Standardization for Quantification and Harmonization of Large-Scale Metabolomics</a>. <strong><em>Analytical Chemistry</em></strong>. 2020 Jul 7;92(13):8836-8844</li>
                <li><strong>Chunyu Ma</strong>, Pamela Madden, Paul Gontarz, Ting Wang and Bo Zhang, <a target='_blank' href="https://pubmed.ncbi.nlm.nih.gov/pubmed/31001319">FeatSNP: An Interactive Database for Brain-Specific Epigenetic Annotation of Human SNPs</a>. <strong><em>Frontiers in Genetics</em></strong>. 2019 Feb 2;10:262</li>
                <li>Xin Hu, Joshua Chandler, Soojin Park, Ken H. Liu, Jolyn Fernandes, Michael Orr, Matthew Ryan Smith, <strong>Chunyu Ma</strong>, Sang-moo Kang, Karan Uppal, Dean Jones, Young-Mi Go, <a target='_blank' href="https://www.ncbi.nlm.nih.gov/pubmed/30529385">Low-dose cadmium disrupts mitochondrial citric acid cycle and lipid metabolism in mouse lung</a>. <strong><em>Free Radical Biology and Medicine</em></strong>. 2019 Feb 1;131:209-217</li>
                <li>Sarah H. Stout, Ganesh M. Babulal, <strong>Chunyu Ma</strong>, David B. Carr, Denise M. Head, Elizabeth A. Grant, Monique M. Williams, David M. Holtzman, Anne M. Fagan, John C. Morris, Catherine M. Roe, <a target='_blank' href="https://www.ncbi.nlm.nih.gov/pubmed/29328928">Driving cessation over a 24-year period: Dementia severity and cerebrospinal fluid biomarkers</a>. <strong><em>Alzheimer's & Dementia</em></strong>. 2018 May;14(5):610-616</li>
                <li>Matheus B. Victor, Michelle Richner, Hannah E. Olsen, Seong Won Lee, Alejandro M. Monteys, <strong>Chunyu Ma</strong>, Christine J. Huh, Bo Zhang, Beverly L. Davidson, X. William Yang & Andrew S. Yoo, <a href="https://www.ncbi.nlm.nih.gov/pubmed/29403030">Striatal neurons directly converted from Huntington's disease patient fibroblasts recapitulate age-associated disease phenotypes</a>. <strong><em>Nature Neuroscience</em></strong>. 2018 Mar;21(3):341-352</li>
                <li>Karan Uppal, <strong>Chunyu Ma</strong>, Young-Mi Go, Dean P Jones, <a  target='_blank' href="https://www.ncbi.nlm.nih.gov/pubmed/29069296">xMWAS: a data-driven integration and differential network analysis tool</a>. <strong><em>Bioinformatics</em></strong>. 2018 Feb 15;34(4):701-702</li>
                <li>Sanne Nygaard, Haofu Hu, Cai Li, Morten Schiøtt, Zhensheng Chen, Zhikai Yang, Qiaolin Xie, <strong>Chunyu Ma</strong>, Yuan Deng, Rebecca B. Dikow, Christian Rabeling, David R. Nash, William T. Wcislo, Seán G. Brady, Ted R. Schultz, Guojie Zhang & Jacobus J. Boomsma, <a target='_blank' href="https://www.ncbi.nlm.nih.gov/pubmed/27436133">Reciprocal genomic evolution in the ant-fungus agricultural symbiosis</a>. <strong><em>Nature Communication</em></strong>. 2016 Jul 20;7:12233</li>
            </ol>
        </div>
    </div>
    <div>
        <h2 id="employment"><i class="fa fa-university"></i> Internship & Employment</h2>
        <dl>
            <dt>05/2020-Present</dt><dd><strong>Research Assistant</strong>, David Koslicki's Laboratory, Penn State University, State Colledge, PA, USA</dd>
            <dt>02/2017-04/2019</dt><dd><strong>Bioinformatician</strong>, Clinical Biomarkers Laboratory, Emory University, Atlanta, GA, USA</dd>
            <dt>06/2016-08/2016</dt><dd><strong>Summer Intern</strong>, Knight Alzheimer's Disease Research Center, St. Louis, MO, USA</dd>
            <dt>10/2015-12/2016</dt><dd><strong>Research Assistant</strong>, Center of Regenerative Medicine, Washington University in St. Louis, St. Louis, MO, USA</dd>
            <dt>11/2013-06/2014</dt><dd><strong>Bioinformatics Analyst</strong>, China National GeneBank, Beijing Genomics Institute, Shenzhen, China</dd>
        </dl>
    </div>

    <div>
        <h2 id="proskills"><i class="fa fa-book"></i> Professional Skills</h2>
        <div class='block1'>
            <h3>Computer Skills</h3>
            <ul>
                <p><strong>Computer languages</strong>: Python (including Cpython & Flask), C, R (including R shiny), Javascript, HTML, CSS, MATLAB</p>
                <p><strong>Other skills</strong>: Linux, MongoDB, SQL, Neo4j, AWS, Google Could</p>
            </ul>
        </div>
        <div class='block1'>
            <h3>Machine Learning Skills</h3>
            <ul>
                <p><strong>Frameworks</strong>: sklearn, pytorch, tensorflow</p>
                <p><strong>Models: </strong>: Basic ML models (e.g. SMV, RF), Graph embedding models (e.g. TransE, TransR, RotatE, DistMult), GNN models (e.g. GCN, GAT, RGCN), RL models (e.g. Q-learning, A2C) </p>
            </ul>
        </div>
        <div class='block1'>
            <h3>Analytical Skills</h3>
            <ul>
                <p><strong>Bioinformatics</strong>: RNA-seq analysis, ChIP-seq analysis, metabolomics analysis (LC-MS)</p>
                <p><strong>Biostatistics</strong>: survival analysis</p>
            </ul>
        </div>
    </div>

    <div>
        <h2 id="software-development"><i class="fa fa-gears"></i> Application Development Products</h2>
        <div class='block1'>
            <h3 id="contributions-to-open-source-libraries"><strong>Bioinformatics Database</strong></h3>
            <dl>
                <dt><a target='_blank' href="http://featsnp.org">FeatSNP</a></dt><dd>An interactive database for brain-specific epigenetic annotation of human SNPs</dd>
                <dt style='margin-top: 5px;'><a target='_blank' href="https://kuppal.shinyapps.io/pycdb">PyCDB</a></dt><dd style='margin-top: 5px;'>A phytochelatin database</dd>
            </dl>
        </div>
        <div class='block1'>
            <h3 id="contributions-to-open-source-libraries"><strong>Contributions to other open source applications</strong></h3>
            
            <dl>
                <dt style='margin-top: 5px;'><a target='_blank' href="https://arax.ncats.io">ARAX</a></dt><dd style='margin-top: 5px;'>A knowledge-graph-based computational reasoning tool for answering biomedical translational questions</dd>
                <dt style='margin-top: 5px;'><a target='_blank' href="https://kuppal.shinyapps.io/xmwas">xMWAS</a></dt><dd style='margin-top: 5px;'>An R shiny app for data integration, network visualization, clustering, and differential network analysis of up to four datasets from omics platforms</dd>
            </dl>
        </div>
    </div>

</body>
</html>
